playground/terraform/output.tf

```diff
@@ -54,26 +54,38 @@ output "docker-repository-root" {
   value = module.infrastructure.docker-repository-root
 }
 
-output "go-server-url" {
-  value = module.applications.go-server-url
-}
+#output "go-server-url" {
```

Comment:
maybe we should delete all commented lines to simplify review?

Reply:
Done